src/CSharp/MetadataWebApi/MetadataWebApi/MetadataWebApi.csproj

```diff
@@ -1,118 +1,23 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
needed to add xmlns="http://schemas.microsoft.com/developer/msbuild/2003" for vs 2015 to load the solution whithout complaining